[firmware] we don't check returncode of the imagebuilder

If one profile (e.g. ubnt) fails, the imagebuilder doesn't fail in all cases.
We need to check the return code for every imagebuilder's build.

Doesn't work here either. Besides lots of `Ignoring feed 'routing' - index missing` I get `Makefile:1227: recipe for target 'modules' failed`. For a full ouput see [here](http://paste.debian.net/hidden/95443aa2/).

--- a/Makefile
+++ b/Makefile
@@ -110,7 +110,7 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	    PACKAGES_FILE_ABS="$$PACKAGES_PATH/$$PACKAGES_FILE.txt"; \
 	    PACKAGES_LIST=$$(grep -v '^\#' $$PACKAGES_FILE_ABS | tr -t '\n' ' '); \
 	    $(UMASK);\
-	    $(MAKE) -C $(IB_BUILD_DIR)/$(IB_DIR) image PROFILE="$$PROFILE" PACKAGES="$$PACKAGES_LIST" BIN_DIR="$(IB_BUILD_DIR)/$(IB_DIR)/bin/$$PACKAGES_FILE"; \
+	    $(MAKE) -C $(IB_BUILD_DIR)/$(IB_DIR) image PROFILE="$$PROFILE" PACKAGES="$$PACKAGES_LIST" BIN_DIR="$(IB_BUILD_DIR)/$(IB_DIR)/bin/$$PACKAGES_FILE" || exit 1; \
 	  done; \
 	done
 	mkdir -p $(FW_TARGET_DIR)

